CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Unreleased
 
--
+- [added] `admin.projectManagement().listAppMetadata()` method to list the app summary of up to 100
```

Comment:
@nbegley, I just want to make sure that the update log looks good

Reply:
LGTM